fabric-convention-tags-v2/src/main/java/net/fabricmc/fabric/api/tag/convention/v2/ConventionalEnchantmentTags.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.tag.convention.v2;
+
+import net.minecraft.enchantment.Enchantment;
+import net.minecraft.registry.tag.TagKey;
+
+import net.fabricmc.fabric.impl.tag.convention.v2.TagRegistration;
+
+public final class ConventionalEnchantmentTags {
+	private ConventionalEnchantmentTags() {
+	}
+
+	/**
+	 * A tag containing enchantments that increase the amount or
+	 * quality of drops from blocks, such as {@link net.minecraft.enchantment.Enchantments#FORTUNE}.
+	 */
+	public static final TagKey<Enchantment> INCREASES_BLOCK_DROPS = register("increase_block_drops");
+	/**
+	 * A tag containing enchantments that increase the amount or
+	 * quality of drops from entities, such as {@link net.minecraft.enchantment.Enchantments#LOOTING}.
+	 */
+	public static final TagKey<Enchantment> INCREASES_ENTITY_DROPS = register("increase_entity_drops");
+	/**
+	 * For enchantments that increase the damage dealt by an item.
+	 */
+	public static final TagKey<Enchantment> WEAPON_DAMAGE_ENHANCEMENT = register("weapon_damage_enhancement");
+	/**
+	 * For enchantments that increase movement speed or otherwise benefit the entity wearing armor enchanted with it.
```

Comment:
The "otherwise" part is very ambiguous; for example, does an armor that gives players villager trade discount count as one?

Reply:
I am not sure what the "otherwise benefit" is referring to as I would assume this tag is only for speed boosting stuff. This tag and Javadoc was added by Dex last year.
![image](https://github.com/FabricMC/fabric/assets/40846040/9799f77b-9e2a-48e6-9b69-4b0c2598421a)

@dexman545 Should I remove the "otherwise benefit" part or was there another kind of speed boosting that you were trying to include in this javadoc?

Reply:
The vast majority of this module is from me since I wrote it. Biome tags are the exception (they're mostly from you, grunt :P), with some renames. These tags date back to when the module was written.

The enchantment tags are a bit iffy as there weren't any precedents I could find.

In this case, I was considering things like frost walker or the auto step enchantment some mods have.

The damage tag was considering the various protections or a knockback resistance enchantment.

Reply:
The tag wasn't just for speed boosts. Maybe these should be broken up/have sub categories? `ENTITY_SPEED_ENHANCEMENT` and `ENTITY_AUXILLARY_MOVEMENT_ENCHANCEMENT`?

Reply:
Would `ENTITY_AUXILLARY_MOVEMENT_ENCHANCEMENT` have Jump Boost and frost walker? Yeah breaking it up might help make it clear that speed is its own thing. I'll do that for now and we can adjust it some more as we go along

Reply:
I would say yes to those being there.